spec/dummy/app/models/course.rb

```diff
@@ -72,4 +72,8 @@ def self.ransackable_attributes(auth_object = nil)
   def self.ransackable_associations(auth_object = nil)
     ["links"]
   end
+
+  def attendees
```

Comment:
Let's leave a comment for why this is here (testing purposes...). And for which resource.